src/components/Explore/panels/TraceViewPanelScene.tsx

```diff
@@ -23,6 +27,47 @@ export interface TracePanelState extends SceneObjectState {
 }
 
 export class TraceViewPanelScene extends SceneObjectBase<TracePanelState> {
+  private async getTraceErrorMessage(errors: DataQueryError[], traceId: string): Promise<string> {
+    const errorMessage = errors?.[0]?.message || '';
+    const status = errors?.[0]?.status;
+
+    if (status === 404 || errorMessage.toLowerCase().includes('not found')) {
```

Comment:
Not sure why I'm getting 400, not 404 🤔 

<img width="1445" height="772" alt="Image" src="https://github.com/user-attachments/assets/d7f67afb-ab3b-460a-b313-1237a64ffb99" />